src/styles/button.scss

```diff
@@ -0,0 +1,23 @@
+.card__button {
+  width: 166px;
+  height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. Use padding instead

Reply:
But in previous comment was said that I need to delete those padding...